Tools/ApplicationDebugger/jacobi/jacobi-bugged.vcxproj.user

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
+    <LocalDebuggerCommandArguments>cpu</LocalDebuggerCommandArguments>
+    <DebuggerFlavor>WindowsLocalDebugger</DebuggerFlavor>
+    <RemoteDebuggerCommandArguments>gpu</RemoteDebuggerCommandArguments>
+    <RemoteDebuggerEnvironment>CL_CONFIG_USE_NATIVE_DEBUGGER=1</RemoteDebuggerEnvironment>
```

Comment:
Should this be defined for the local env, too?

Reply:
This should be defined only for local environment, thanks!